### PR TITLE
Support custom character icons / edition logos again with opt-in

### LINKED ISSUE #74: Feature request: custom role icons / custom script logo
I think there are a couple of graphical bugs with the custom character support in scripts.

The first is custom token images to not appear to resize correctly for the player tokens, but are fine for reminder tokens.
image
If it helps the images I have been working with are https://raw.githubusercontent.com/davotronic5000/ChristmasBrewing/main/Santa-Sack.png?raw=true and https://raw.githubusercontent.com/davotronic5000/ChristmasBrewing/main/Grinch.png?raw=true.  It might just be that there are some sizing requirements needed for the images used.

The second is if a json file is uploaded by a player that includes a custom logo, it displays correctly, but if you then join a STs session running the same script the custom logo is replaced with the default "Custom Script" logo.

### PR DESCRIPTION
**Pull request by bra1n**
I've added support for sending custom edition characters through the live session again, as well as showing custom edition logos for players, after they opt in through the session menu.
This closes #74

### CODE REVIEW

**Comment by bra1n** [comment]
@davotronic5000 would you like to review this PR before I merge it?

**Comment by bra1n** [comment]
Sorry for the messy commit history. 😑

**Review by davotronic5000 — CHANGES_REQUESTED** [review]
Code looks great, only issus I have found is by deafult now the Story teller cannot see the custom images from a script they have uploaded, unless they have the image optin already saved from a player session.

File: src/components/Player.vue [hunk 1]
```diff
@@ -165,8 +165,13 @@
         <span
           class="icon"
           :style="{
-            backgroundImage: `url(${reminder.image ||
-              require('../assets/icons/' + reminder.role + '.png')})`
+            backgroundImage: `url(${
+              reminder.image && grimoire.isImageOptIn
```
[the remark this answers is not in the document]
Comment (bra1n) [reply src/components/Player.vue:169]: Ooh, really good catch! Thanks, I totally missed that after enabling it for the host. I'll show the image opt-in in the Grimoire tab now, whether online or offline. (I've just realized that you don't even need to be in a live session to load homebrew scripts with custom images)

**Pull request merged**